src/main/java/de/lukweb/justmail/socket/ImapSocketHandler.java

```diff
@@ -65,16 +65,16 @@ public void run() {
                 }
                 cache += (char) read;
                 if (!(cache.endsWith("\n\r") || cache.endsWith("\n"))) continue;
-                /* if (session.getCallback() != null) {
+                if (session.getCallback() != null) {
                     boolean breakCB = session.getCallback().callback(cache);
                     if (breakCB) session.setCallback(null);
                     cache = "";
                     continue;
-                } */
-                String[] split = cache.split(" ", 1);
```

Comment:
We have to split the string only once, because we want to separate the tag from the string. And if the string was split once there are two different strings in this array, that's why it worked before.